staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -131,6 +131,8 @@ type Config struct {
 	OpenAPIConfig *openapicommon.Config
 	// SwaggerConfig will be used in generating Swagger spec. This is nil by default. Use DefaultSwaggerConfig for "working" defaults.
 	SwaggerConfig *swagger.Config
+	// RESTOptionsGetter is used to construct "normal" RESTStorage types
```

Comment:
Please add "This has a reasonable default for most apiservers." or something like that.